deploy_and_run_integration_tests.sh

```diff
@@ -168,20 +197,18 @@ export VIRTUAL_ENV_DISABLE_PROMPT="something"
 virtualenv "${temp_dir}"
 source ${temp_dir}/bin/activate;
 trap clean_up EXIT
+if [[ -n "${run_unit_tests}" ]]; then
```

Comment:
a bit counterintuitive: the script is called deploy_and_run_integration test, but it runs unit test too.

Reply:
I changed the script name to deploy_and_run_tests